### PR TITLE
Feat/connect

### PR DESCRIPTION
**Pull request by churchill-zhang**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

File: framework/android/src/main/java/com/tencent/mtt/hippy/HippyEngineManagerImpl.java [hunk 1]
```diff
@@ -783,29 +761,31 @@ private void initVfsManager() {
         private void initDevtoolsManager() {
             String localCachePath = getGlobalConfigs().getContext().getCacheDir()
                     .getAbsolutePath();
-            mDevtoolsManager.create(mWorkerManagerId, localCachePath,
-                    getDevSupportManager().createDebugUrl(mServerHost));
+//            mDevtoolsManager.create(mWorkerManagerId, localCachePath,
+//                    getDevSupportManager().createDebugUrl(mServerHost));
         }
```
Comment (lavnFan) [inline framework/android/src/main/java/com/tencent/mtt/hippy/HippyEngineManagerImpl.java:766]: 没恢复

**Review by medns — CHANGES_REQUESTED** [review]
need to be change

**Review by siguangli — APPROVED** [review]

**Review by ozonelmy — APPROVED** [review]

**Review by ilikethese — APPROVED** [review]
TL;DR;

**Review by lavnFan — APPROVED** [review]
lgtm

**Review by medns — APPROVED** [review]
LGTM
